apps/messaging/views.py

```diff
@@ -20,6 +20,7 @@ class ChatView(TemplateView):
     # form_class: Type[ChatForm] = ChatForm
 
     def get(self, request: WSGIRequest, **kwargs: Dict[str, Any]) -> HttpResponse:
+        context = self.get_context_data()
```

Comment:
I use this in general, which means inheritance/mixins can add stuff to the context, and then we can add to it here...